Client/src/Components/TabPanels/Account/ProfilePanel.jsx

```diff
@@ -116,13 +117,23 @@ const ProfilePanel = () => {
   // Resets picture-related states and clears interval
   const removePicture = () => {
     errors["picture"] && clearError("picture");
+    setFile({ delete: true });
     clearInterval(intervalRef.current); // interrupt interval if image upload is canceled prior to completing the process
     setProgress({ value: 0, isLoading: false });
   };
 
+  // Opens the picture update modal
+  const openPictureModal = () => {
+    setIsOpen("picture");
```

Comment:
I see why this is done, we need to distinguish between which operation opened the modal, but the `isModalOpen` method is kind of confusing as a result :thinking: 

We might want to add a TODO here to revisit this and think about a cleaner way of handling this, but no big deal for now

Reply:
The alternative is we could drop `isModalOpen` entirely and use `isOpen === value` directly, if that makes it any less confusing. I'm open to suggestions. I wasn't really sure how else I could implement it. It's a little tricky because we're dealing with two modals that are dependent on one another in the sense that only one can be active at a time.

Reply:
What about just setting the open value of one to be the inverse of the other?

That way they can never be open at the same time?

Reply:
Wouldn't that just open the 2nd modal when you close the 1st and vice versa 😂 ?

Reply:
I hearby retract my previous comment and slink away in shame 🤣

Reply:
I think we can leave it for now, it's not critical. Let's keep it in mind though in case a better solution comes to us